packages/core/nuxt-theme-module/theme/pages/CheckoutV2/Shipping.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div>
+    <SfHeading
+      :level="3"
+      title="Shipping"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <ShippingForm
+      :isSaving="isSaving"
+      :handleShippingMethodSubmit="handleShippingMethodSubmit"
+      :handleShippingAddressSubmit="handleShippingAddressSubmit"
+      @stepSubmit="handleStepSubmit"
+    />
+  </div>
+</template>
+
+<script>
+import {
+  SfHeading
+} from '@storefront-ui/vue';
+import { useShipping, useShippingMethod } from '@vue-storefront/commercetools';
+import { onSSR } from '@vue-storefront/core';
+import { reactive } from '@vue/composition-api';
+import ShippingForm from '../../components/Checkout/ShippingForm';
+
+export default {
+  name: 'Shipping',
+  components: {
+    SfHeading,
+    ShippingForm
+  },
+  setup(_, context) {
+    const {
+      save: saveShipping,
+      load: loadShipping,
+      error: shippingError
+    } = useShipping();
+    const {
+      save: saveShippingMethod,
+      load: loadShippingMethod,
+      error: shippingMethodError
+    } = useShippingMethod();
+
+    const isSaving = reactive({
+      details: false,
+      method: false
+    });
+
+    onSSR(async () => loadShipping());
```

Comment:
Missing `await`

Reply:
Well, this method still returns promise `() => loadShipping()`. I decided to add `async` at the beginning to make developers conscious `loadShipping` is async function. I see no benefit from `awaiting` it

Reply:
When a new method will be added to this function, it will be very easy to forget to add `await` before `loadShipping()`.

Reply:
Agree